packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -41,7 +41,7 @@ Generated by [AVA](https://ava.li).
     > Available plugins␊
     ␊
     > Loading plugins␊
-       - netlify-plugin-contextual-env 0-3-0 from netlify.toml␊
+       - netlify-plugin-contextual-env 0-3-0 from netlify.toml (latest 1.0.0, expected 1.0.0, compatible 1.0.0)␊
```

Comment:
Wondering if all the `latest`, `expected` and `compatible` versions on this snapshot are expected to be `1.0.0`? Or is this a consequence of the string replacemente in our snapshots?

Reply:
It is due to the snapshot normalization.
I will create a second PR to fix it, so we remove that normalization :+1:

Reply:
https://github.com/netlify/build/issues/2698